src/ZkFold/Cardano/Plonk.hs

```diff
@@ -177,3 +179,12 @@ instance (KnownNat n, KnownNat (3 * n), KnownNat ((4 * n) + 6)) => CompatibleNon
     nipSetupTransform = mkSetup
     nipInputTransform = mkInput
     nipProofTransform = mkProof
+
+untypedVerifyPlonk :: SetupBytes -> BuiltinData -> BuiltinData -> BuiltinUnit
```

Comment:
Suggest adding INLINABLE pragma.  Also, type signature of `untypedVerifyPlonk` should be `SetupBytes -> BuiltinData -> BuiltinUnit` .  For example, see my comments for file `src/ZkFold/Cardano/Scripts/PlonkVerifier.hs` .